migrations/versions/9485c5134ac0_.py

```diff
@@ -0,0 +1,36 @@
+"""empty message
+
+Revision ID: 9485c5134ac0
+Revises: b82ed93cfc3f
+Create Date: 2021-03-09 23:05:01.389901
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '9485c5134ac0'
+down_revision = 'b82ed93cfc3f'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.drop_table('result')
+    # ### end Alembic commands ###
+
+
+def downgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('result',
+    sa.Column('id', sa.INTEGER(), nullable=False),
+    sa.Column('election_id', sa.INTEGER(), nullable=False),
+    sa.Column('candidate_id', sa.INTEGER(), nullable=False),
+    sa.Column('total_votes', sa.VARCHAR(), nullable=True),
+    sa.ForeignKeyConstraint(['candidate_id'], ['candidate.id'], name='fk_result_candidate_id_candidate', ondelete='CASCADE'),
+    sa.ForeignKeyConstraint(['election_id'], ['election.id'], name='fk_result_election_id_election', ondelete='CASCADE'),
+    sa.PrimaryKeyConstraint('id', name='pk_result')
+    )
```

Comment:
I dropped and created the table because the changes didn't reflect in the migration

Reply:
I still can't find the migration for the Results table bro. I will add it dw